$OpenBSD: patch-texk_web2c_xetexdir_xetex_ch,v 1.1 2011/11/23 12:00:45 edd Exp $

Reliability fix from texlive 2011 stable branch. Off by one causes segfault:
http://tug.org/pipermail/tldistro/2011q3/000137.html

--- texk/web2c/xetexdir/xetex.ch.origy	Tue Apr 19 14:44:09 2011
+++ texk/web2c/xetexdir/xetex.ch	Tue Sep  6 14:57:22 2011
@@ -328,7 +328,7 @@ for i:=@'177 to @'377 do xchr[i]:=i;
 {Initialize enc\TeX\ data.}
 for i:=0 to 255 do mubyte_read[i]:=null;
 for i:=0 to 255 do mubyte_write[i]:=0;
-for i:=0 to 128 do mubyte_cswrite[i]:=null;
+for i:=0 to 127 do mubyte_cswrite[i]:=null;
 mubyte_keep := 0; mubyte_start := false;
 write_noexpanding := false; cs_converting := false;
 special_printing := false; message_printing := false;
